tests/helper_functions.py

```diff
@@ -65,7 +65,7 @@ def pyfunc_serve_and_score_model(
     """
     env = dict(os.environ)
     env.update(LC_ALL="en_US.UTF-8", LANG="en_US.UTF-8")
-    scoring_cmd = ['mlflow', 'pyfunc', 'serve', '-m', model_uri, "-p", "0"]
+    scoring_cmd = ['mlflow', 'models', 'serve', '-m', model_uri, "-p", "0"]
```

Comment:
btw, we may consider just making it `mlflow serve` and `mlflow predict` instead of being within models

Reply:
Oh I thought we have decided on having this inside models? Either way, it should be a simple change on the cli side.